tox.ini

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 [tox]
-envlist = pep8, pylint, py27, jenkins
+envlist = py27, pep8, pylint
```

Comment:
added .eggs to .gitignore, .dockerignore, and the pep8 exclude list.  I think it may have been packaged into the docker container, and hopefully will go away when we cut the official release.